assets/stylesheets/kitten/components/form/_location-input.scss

```diff
@@ -0,0 +1,174 @@
+/// Location input
+///
+/// @group form
+///
+/// @example scss - usage
+///
+///   @include k-LocationInput;
+///
+/// @example html
+///
+///   <div class="k-LocationInput">
+///     <div class="k-LocationInput__label">…</div>
+///     <div class="k-LocationInput__icon">
+///       <svg />
+///     </div>
+///     <input class="k-LocationInput__group
+///                   k-LocationInput__input
+///                   k-LocationInput__autocomplete" />
+///   </div>
+
+@mixin k-LocationInput {
+  $font: 'source-sans';
+  $label-font: 'source-sans-semi-bold';
+  $font-size: -1;
+  $line-height: 1.3;
+  $height: k-px-to-rem(50px);
+
+  $icon-margin: k-px-to-rem(17px);
+
+  // Color autocomplete
+  $color-mainText: map-get($k-colors, 'font-1');
+  $color-secondaryText: map-get($k-colors, 'font-2');
+  $border-autocomplete: map-get($k-colors, 'line-2');
+
+  // Base
+  $color: map-get($k-colors, 'font-1');
+  $border: map-get($k-colors, 'line-2');
+  $background: map-get($k-colors, 'background-1');
+
+  // Focus
+  $border-focus: map-get($k-colors, 'primary-1');
+
+  // Disabled
+  $color-disabled: #fff;
+  $border-disabled: #fff;
+  $background-disabled: map-get($k-colors, 'line-2');
+
+  // Placeholder
+  $color-placeholder: map-get($k-colors, 'font-2');
+
+  // Is-valid
+  $color-is-valid: map-get($k-colors, 'valid');
+  $border-is-valid: map-get($k-colors, 'valid');
+
+  // Is-error
+  $color-is-error: map-get($k-colors, 'error');
+  $border-is-error: map-get($k-colors, 'error');
```

Comment:
Si on prévoit des couleurs d'erreur, peut-être qu'il faut que notre composant React puisse avoir cette classe, non ? Par exemple avoir une propriété `<LocationInput error />`.

(Sinon on peut juste retirer la classe `is-error` en attendant.)